OpenRA.Game/Sound/Sound.cs

```diff
@@ -45,10 +45,12 @@ public sealed class Sound : IDisposable
 		ISound video;
 		MusicInfo currentMusic;
 		Dictionary<uint, ISound> currentSounds = new Dictionary<uint, ISound>();
+		public bool DummyEngine;
```

Comment:
Since this is set in the ctor, you can make it `readonly`, or (maybe even better) make it a `{ get; private set; }` property.

Reply:
Done, thanks for the tip!